### PR TITLE
Fishingmap/load categories by spreadsheet

### PR DESCRIPTION
**Pull request by j8seangel**
Show only categories which are in: 
- https://github.com/GlobalFishingWatch/frontend/blob/1b93b35250920ce4149ab523a26d79e3c885ff5d/applications/fishing-map/src/data/workspaces.ts#L16 
- and in the spreadsheet, checking if the category has content too 

to be able to generate categories with only marine-reserves categories, or maybe country portals for the future
image

### CODE REVIEW

**Comment by j8seangel** [comment]
See https://globalfishingwatch.atlassian.net/browse/MR-185 to get the `spreadsheetId` to test

File: applications/fishing-map/src/features/workspaces-list/WorkspacesList.tsx [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
 
 function WorkspacesList() {
   const { t } = useTranslation()
-  const dispatch = useDispatch()
   const locationCategory = useSelector(selectLocationCategory)
   const userFriendlyCategory = locationCategory.replace('-', ' ')
   const highlightedWorkspaces = useSelector(selectCurrentHighlightedWorkspaces)
   const highlightedWorkspacesStatus = useSelector(selectHighlightedWorkspacesStatus)
-  const validCategory = Object.values(WorkspaceCategories).includes(locationCategory)
+  const validCategory = useSelector(isValidLocationCategory)
```
Comment (rdgfuentes) [inline applications/fishing-map/src/features/workspaces-list/WorkspacesList.tsx:21]: Great moving this to a selector 👏

File: applications/fishing-map/src/features/workspaces-list/workspaces-list.selectors.ts [hunk 2]
```diff
@@ -23,6 +23,24 @@ export const selectCurrentWorkspaces = createSelector(
   }
 )
 
+export const selectAvailableWorkspacesCategories = createSelector(
```
Comment (rdgfuentes) [inline applications/fishing-map/src/features/workspaces-list/workspaces-list.selectors.ts:26]: 👍

**Review by rdgfuentes — APPROVED** [review]
LGTM. It'd be great to add some tests, maybe a snapshot test for the `CategoryTabs` and some unit tests for the selector `selectAvailableWorkspacesCategories`

**Pull request merged**